doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1705,13 +2053,16 @@ performance issue.
 
    This flag enables and disables the profiler.
 
-.. cmd:: Show Ltac Profile
+.. cmd:: Show Ltac Profile {? {| CutOff @int | @string } }
 
-   Prints the profile
+   Prints the profile.
 
-   .. cmdv:: Show Ltac Profile @string
+   :n:`CutOff @int`
+      to be added
```

Comment:
I'm pretty sure this is something that @JasonGross can explain (he may have been the one to introduce it in fact).

Reply:
By default, `Show Ltac Profile` will hide any tactics which account for (when descendants are included) less than 2% of the total time.  The number `2` can be replaced by any other non-negative integer `n` by specifying `Show Ltac Profile CutOff n`.  (`Show Ltac Profile` is the same as `Show Ltac Profile CutOff 2`, unless you've passed (apparently undocumented!) option `-profile-ltac-cutoff` with another number (which is then used instead).